.github/workflows/publish_release.yml

```diff
@@ -124,11 +124,17 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           branch: "1.x"
 
+      - name: Prepare changelog release notes
+        id: changelog-notes
+        run: |
+          notes="$(echo "${{ steps.notes.outputs.release-notes }}" | sed "s/^## /### /")"
+          echo "release-notes=$notes" >> $GITHUB_OUTPUT
+
       - name: Update Changelog
         uses: stefanzweifel/changelog-updater-action@v1
         with:
-          release-notes: ${{ steps.notes.outputs.release-notes }}
-          latest-version: "v${{ github.event.release.tag_name }}"
+          release-notes: ${{ steps.changelog-notes.outputs.release-notes }}
+          latest-version: "v${{ steps.bump-version.outputs.version }}"
```

Comment:
This was using the wrong value and only seeing `v` rather than `vx.x.x`.

See: https://github.com/laravel/nightwatch/actions/runs/17087746462/job/48455309415#step:13:19